packages/theme-dark-extension/style/variables.css

```diff
@@ -424,10 +424,10 @@ all of MD as it is not optimized for dense, information rich UIs.
   --jp-reject-color-active: var(--md-grey-800);
 
   /* File or activity icons and switch semantic variables */
-  --jp-jupyter-icon-color: var(--md-orange-900);
+  --jp-jupyter-icon-color: var(--md-orange-700);
```

Comment:
I see that this is only changed in dark theme and not the other. Is the intention to have two different colours depending on theme? If this is intended, please update the PR description to make this explicit for future reference.

As per [Jupyter brand guide](https://github.com/jupyter/design/blob/master/brandguide/brand_guide.pdf) the colour should be `#F37626` which is in between `--md-orange-700` (`#f57c00`) and `--md-orange-900` (`#e65100`) (but closer to 700). I am **not** requesting changes, just wanted to make sure you are aware that we do not currently comply with own brand guidelines. Suggestions on how to fix that are welcome, though it may be better discussed in a new issue :)

Reply:
I realized that the color warn# and brand# are actually not the same in the light and the dark themes. As the icon was using those - this change is to avoid introducing a change. But you are making a good point, it sound better to set the official color for the Jupyter icon.

Reply:
I can update the description. Indeed most of the changes are for the dark theme except a correction for notebook icon color in light theme. Initially (before PR #13279) I thought from `variable.css` file that the color for jupyter and notebook were defined through :

-  `--jp-warn-color0: var(--md-orange-900)` for the light theme

-  `--jp-warn-color0: var(--md-orange-700)` for the dark theme.

I may be missing some transformation that turns jupyter color from `--md-orange-900` to  `#F37626` (for light theme).

Reply:
Thanks! Just for context, about a year ago we changed `--jp-warn-color0` to use a darker colour to improve contrast of buttons. Change of the logo colour was not intended, but just happened because they were using the same variable. All I am saying is that we were not compliant with the brand colour for some time now, but your PR separating these variables did enable us to fix it so it is a good time to address it one way or the other).

Reply:
> Thanks! Just for context, about a year ago we changed `--jp-warn-color0` to use a darker colour to improve contrast of buttons. Change of the logo colour was not intended, but just happened because they were using the same variable. All I am saying is that we were not compliant with the brand colour for some time now, but your PR separating these variables did enable us to fix it so it is a good time to address it one way or the other).

Ok thanks for giving me these elements of context (that I did not have). So I am even more happy to give back the "true" branc color to these icons :-)

Reply:
I found this in base.css of the launcher 
```
.jp-LauncherCard[data-category='Notebook'] .jp-LauncherCard-noKernelIcon {
  /* This color is copied from the notebook icon. */
  color: #ef6c00;
}
```
I will propose to change it to the #F37626. If you know other places where this may be corrected. Please let me know.